dice.js

```diff
@@ -0,0 +1,26 @@
+
+
+
+function rollDice(rollN){
+    let num1 = Math.round(Math.random() * 6)
```

Comment:
Great solution; you create a random number within a range starting from 0 to 6 but we as we know dice does not have 0 so we need to enhance the range.
```Math.round(Math.random() * (max - min) + min);```